restlicsharpclient/restliclient/transport/DefaultTransportClient.cs

```diff
@@ -59,7 +59,7 @@ private HttpRequestMessage ConstructHttpRequestMessage(HttpRequest httpRequest)
         {
             HttpRequestMessage httpRequestMessage = new HttpRequestMessage()
             {
```

Comment:
I just noticed this weird syntax, in C#, we are not passing parameters inside HttpRequestMessage(method, requestUri), instead can use this closure like syntax to arbitrarily switch the parameter order?